consent/strategy_default_test.go

```diff
@@ -27,7 +27,7 @@ import (
 	"net/http/httptest"
 	"testing"
 
-	jwtgo "github.com/dgrijalva/jwt-go"
+	jwtgo "github.com/ory/fosite/token/jwt"
 	"github.com/stretchr/testify/require"
 
 	"github.com/ory/fosite/token/jwt"
```

Comment:
I think it would be nicer to have just single import here, `"github.com/ory/fosite/token/jwt"`, no need for both `"github.com/ory/fosite/token/jwt"` and `jwtgo "github.com/ory/fosite/token/jwt"`.

Reply:
Also, grouping of imports is now wrong.